integrations/snyk/webhook_processors/issue_webhook_processor.py

```diff
@@ -0,0 +1,51 @@
+import hashlib
+import hmac
+import json
+from initialize_client import init_client
+from kinds import Kinds
+from port_ocean.core.handlers.port_app_config.models import ResourceConfig
+from port_ocean.core.handlers.webhook.abstract_webhook_processor import (
+    AbstractWebhookProcessor,
+)
+from port_ocean.core.handlers.webhook.webhook_event import (
+    EventHeaders,
+    EventPayload,
+    WebhookEvent,
+    WebhookEventRawResults,
+)
+from port_ocean.context.ocean import ocean
+
+
+class IssueWebhookProcessor(AbstractWebhookProcessor):
+    def should_process_event(self, event: WebhookEvent) -> bool:
+        signature = event.headers.get("x-hub-signature", "")
+        hmac_obj = hmac.new(
+            ocean.integration_config["webhook_secret"].encode("utf-8"),
+            json.dumps(event.payload, separators=(",", ":")).encode("utf-8"),
+            hashlib.sha256,
+        )
+        expected_signature = f"sha256={hmac_obj.hexdigest()}"
+        return signature == expected_signature
```

Comment:
this code seems repeated, I would suggest creating `SnykBaseWebhookProcessor` where the `should_process_event` will be implemented.